js_modules/dagit/packages/core/src/workspace/VirtualizedAssetRow.tsx

```diff
@@ -0,0 +1,281 @@
+import {gql, useLazyQuery} from '@apollo/client';
+import {Box, Caption, Checkbox, Colors, Icon} from '@dagster-io/ui';
+import * as React from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components/macro';
+
+import {ASSET_NODE_LIVE_FRAGMENT} from '../asset-graph/AssetNode';
+import {AssetLatestRunWithNotices, AssetRunLink} from '../asset-graph/AssetRunLinking';
+import {buildLiveDataForNode} from '../asset-graph/Utils';
+import {ASSET_LATEST_INFO_FRAGMENT} from '../asset-graph/useLiveDataForAssetKeys';
+import {AssetActionMenu} from '../assets/AssetActionMenu';
+import {AssetLink} from '../assets/AssetLink';
+import {ASSET_TABLE_FRAGMENT} from '../assets/AssetTableFragment';
+import {StaleTag} from '../assets/StaleTag';
+import {assetDetailsPathForKey} from '../assets/assetDetailsPathForKey';
+import {AssetTableFragment} from '../assets/types/AssetTableFragment';
+import {AssetViewType} from '../assets/useAssetView';
+import {RepositoryLink} from '../nav/RepositoryLink';
+import {TimestampDisplay} from '../schedules/TimestampDisplay';
+import {HeaderCell, Row, RowCell} from '../ui/VirtualizedTable';
+
+import {LoadingOrNone, useDelayedRowQuery} from './VirtualizedWorkspaceTable';
+import {RepoAddress} from './types';
+import {SingleAssetQuery, SingleAssetQueryVariables} from './types/SingleAssetQuery';
+import {workspacePathFromAddress} from './workspacePath';
+
+const TEMPLATE_COLUMNS = '1.3fr 1fr 1fr 80px';
+const TEMPLATE_COLUMNS_FOR_CATALOG = '76px 1.3fr 1.3fr 1fr 1fr 80px';
```

Comment:
I pulled this down and made sure the latest version looks good in all my states ✅

<img width="1840" alt="image" src="https://user-images.githubusercontent.com/1037212/208520323-fd779b58-e5b4-429d-ae5d-f6ef6d722120.png">